node/index.js

```diff
@@ -77,7 +78,16 @@ function TChannel(options) {
 
     self.peers = Object.create(null);
 
-    self.endpoints = Object.create(null);
+    if (self.options.handleRequest) {
+        self.router = null;
+        self.handleRequest = self.options.handleRequest;
+    } else {
+        self.router = TRouter();
```

Comment:
Perhaps else if options.router, curry handleRequest on options.router. So provide handleRequest, provide router, or fall back to default.